WcaOnRails/app/views/static_pages/organizer_guidelines.erb

```diff
@@ -1,6 +1,73 @@
 <% provide(:title, t("organizer_guidelines.title")) %>
 <div class="container">
   <h1><%= yield(:title)  %></h1>
+
+  <p><%= t('full_organizer_guidelines.description') %></p>
+  <p><%= t('full_organizer_guidelines.disclaimer_html') %></p>
+
+  <p><%= t('full_organizer_guidelines.essential.description') %></p>
+  <div class="list-group">
+    <%= link_to "/edudoc/organizer-guidelines/finding-venue.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.finding-venue') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/orga-team-delegate.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.orga-team-delegate') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/budget.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.budget') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/schedule.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.schedule') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/checklist.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.checklist') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/registration.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.registration') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/venue-setup.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.venue-setup') %>
+    <% end %>
+    <%= link_to "/edudoc/organizer-guidelines/work-during-comp.pdf", class: "list-group-item" do %>
+      <%= ui_icon("file alt") %>
+      <%= t('full_organizer_guidelines.essential.work-during-comp') %>
+    <% end %>
```

Comment:
Given that the items are the same, could you please generate them? Something like this:

```erb
<% %w(finding-venue orga-team-delegate budget schedule checklist registration venue-setup work-during-comp).each do |doc| %>
  <%= link_to "/edudoc/organizer-guidelines/#{doc}.pdf", class: "list-group-item" do %>
    <%= ui_icon("file alt") %>
    <%= t("full_organizer_guidelines.essential.#{doc}") %>
  <% end %>
<% end %>
```

Reply:
I assume this would then go after the "<div class="list-group">"?

Reply:
I assume this would go after < div class="list-group" > ? (spaces added so it show up)

Reply:
It looks to have worked, Travis passed. Though coverage feel, but I don't think that matters?

Reply:
Nevermind, now it says coverage remained the same. So I guess all should be good to go. :)